decidim-initiatives/app/views/decidim/initiatives/admin/answers/edit.html.erb

```diff
@@ -1,3 +1,5 @@
+<% add_decidim_page_title(t("decidim.admin_log.helpers.answers")) %>
```

Comment:
This string is lowercase, can you change it: 
![Screenshot of the string](https://github.com/decidim/decidim/assets/717367/09823732-8163-439c-a406-16e5e6a10288)
 
We're also reusing the string from another unrelated component (why ` admin_log` in Initiatives?), so we should change this to 

```suggestion
<% add_decidim_page_title(t(".title")) %>
```

And add the "Answers" string in the corresponding key of the en.yml

Reply:
This wasn't done